src/structures/MessageEmbed.js

```diff
@@ -7,11 +7,19 @@ const Util = require('../util/Util');
  * Represents an embed in a message (image/video preview, rich embed, etc.)
  */
 class MessageEmbed {
-  constructor(data = {}) {
-    this.setup(data);
+  /**
+   * @param {Object} data The data to build the embed from
+   * @param {boolean} skipValidation A flag to skip field validations
```

Comment:
```suggestion
   * @param {Object} [data={}] The data to build the embed from
   * @param {boolean} [skipValidation=false] A flag to skip field validations
```
both of these parameters are optional